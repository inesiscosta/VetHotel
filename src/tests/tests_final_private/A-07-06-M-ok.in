8
5
h1
t1
nova
3
2
EVERGREEN
0
0
