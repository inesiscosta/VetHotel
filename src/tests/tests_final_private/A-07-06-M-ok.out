Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Workers
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Identificador único do habitat: Identificador único da árvore: Nome da árvore: Idade da árvore: Dificuldade de limpeza da árvore: Tipo de árvore: (CADUCA ou PERENE) Plant Tree: Operação inválida: A árvore 't1' já existe.
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Workers
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: 